dictionary.go

```diff
@@ -566,18 +568,20 @@ func (index indexedColumnIndex) NumPages() int       { return 1 }
 func (index indexedColumnIndex) NullCount(int) int64 { return 0 }
 func (index indexedColumnIndex) NullPage(int) bool   { return false }
 func (index indexedColumnIndex) MinValue(int) Value {
-	min, _ := index.col.Bounds()
+	min, _, _ := index.col.Bounds()
 	return min
 }
 func (index indexedColumnIndex) MaxValue(int) Value {
-	_, max := index.col.Bounds()
+	_, max, _ := index.col.Bounds()
 	return max
 }
 func (index indexedColumnIndex) IsAscending() bool {
-	return index.col.typ.Compare(index.col.Bounds()) < 0
+	min, max, _ := index.col.Bounds()
+	return index.col.typ.Compare(min, max) <= 0
```

Comment:
@lmarburger I believe this escaped the tests you added in https://github.com/segmentio/parquet-go/pull/138, we might not be validating cases with dictionary pages.